BP/common/culture/cultures/00_north_germanic.txt

```diff
@@ -958,7 +958,7 @@
 			"dynn_Agiþring"
 			"dynn_Weohsing"
 			"dynn_Ecgþing"
-			"dynn_Wægmunding"
+			"dynn_WE_gmunding"
```

Comment:
Where do you define dynn_WE_gmunding = dynn_Wægmunding?

Reply:
Paradox has "æ" equal "E_". If you're asking why I only changed that one it's because I decided to leave dynasties for later and just move on and I'd already done that one anyway.

Reply:
Ok, I hadn't understood that was automatic, but then the dynasty file should be updated too (later on).

Reply:
Yeah.

Reply:
> Ok, I hadn't understood that was automatic, but then the dynasty file should be updated too (later on).
Oh wait very much no it's not automatic but like that's the coding standards they'd applied uniformly. We'll still have to add it into loc and stuff later.

Reply:
> > Ok, I hadn't understood that was automatic, but then the dynasty file should be updated too (later on).
> 
> Oh wait very much no it's not automatic but like that's the coding standards they'd applied uniformly. We'll still have to add it into loc and stuff later.

That is what I thought initially, indeed